app/language/map.yaml

```diff
@@ -171,13 +171,13 @@ TIME:
   en: time
```

Comment:
Hey thought you said this file was unneeded - want to take it out?
https://github.com/aberke/check-list/pull/6#discussion-diff-23490635R1